### DISCUSSION TITLE
Clock widget with seconds

### DISCUSSION THREAD

**Discussion by sebastienbb16** [breezy-weather/breezy-weather#1016] [Ideas]
### Describe your idea

Good morning,

 It would be interesting to be able to offer a widget with clock + seconds with Weather.

 THANKS.

Bonjour, 

Il serait intéressant de pouvoir proposer un widget avec horloge + secondes avec Météo.

Merci.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.2 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by gtianw** [comment]
I strongly recommend adding this feature.
If someone doesn't like it, we can add an option that is turned off by default.

**Reply by hao123111230** [reply]
First, I also hope to add this feature. Users who are not used to it can turn it off in their personal preferences. Adding this feature has its benefits and drawbacks. The drawbacks are that it may introduce some bugs, such as a discrepancy with the system time (for example, a difference of one or two seconds), and it could also cause stuttering.

**Comment by papjul** [comment]
While we won't add this feature, you can now make your own widgets using #2089
